src/ui/public/registry/feature_catalogue.js

```diff
@@ -4,7 +4,8 @@ export const FeatureCatalogueRegistryProvider = uiRegistry({
   name: 'featureCatalogue',
   index: ['id'],
   group: ['category'],
-  order: ['title']
+  order: ['title'],
+  filter: value => Object.keys(value).length > 0
```

Comment:
What is value? Is value the registry item? Maybe rename so its more explicit?

Reply:
Looks like your reasoning for having this function is that undefined can be returned when there are license problems. Would it make more sense to just check that the object is not undefined? Checking that there are keys makes it less obvious what the filter is actually filtering out.

Reply:
So what I've observed is that when the function, `fn`,  passed into `SomeRegistry.register(fn)` returns a falsy value, the actual item that makes it into the registry is an empty object, `{}`. Hence this check is written the way it is.

Reply:
Renamed variable in 426d760.